terraform/ec2/amis.tf

```diff
@@ -89,46 +89,137 @@ EOF
 ########################################
 variable "amis" {
   default = {
-    ubuntu16 = {
+    # Ubuntu Distribution
+    ubuntu18 = {
       os_family          = "ubuntu"
-      ami_search_pattern = "ubuntu/images/hvm-ssd/ubuntu-xenial-16.04-amd64-server-*"
-      ami_owner          = "099720109477"
-      ami_id             = "ami-0688ba7eeeeefe3cd"
+      ami_search_pattern = "ubuntu/images/hvm-ssd/ubuntu-bionic-*"
+      owners             = "aws-marketplace"
       ami_product_code   = []
       family             = "debian"
-      arch               = "amd64"
+      arch               = "x86_64"
```

Comment:
The `arm_ubuntu18` object declares an instance type. Why doesn't `ubunutu18`? What is the default?

Reply:
The default values come from [here](https://github.com/aws-observability/aws-otel-test-framework/blob/45ee692bac2ca73baa0132c6fcc4e122dc8cdd3f/terraform/ec2/amis.tf#L21) based on the `family` attribute. For ARM images we override to `t4g.nano `because`t2.micro` is not supported to run ARM binaries

Reply:
We should probably be consistent with `micro` or `nano`.  If we're using `t2.micro` let's also use `t4g.micro` so that RAM and burst performance are equivalent.